projects/packages/videopress/src/class-initializer.php

```diff
@@ -76,5 +76,24 @@ public static function register_oembed_providers() {
 		wp_oembed_add_provider( '#^https?://video.wordpress.com/v/.*#', 'https://public-api.wordpress.com/oembed/?for=' . $host, true );
 		// This is needed as it's not supported in oEmbed discovery
 		wp_oembed_add_provider( '|^https?://v\.wordpress\.com/([a-zA-Z\d]{8})(.+)?$|i', 'https://public-api.wordpress.com/oembed/?for=' . $host, true ); // phpcs:ignore WordPress.WP.CapitalPDangit.Misspelled
+
+		add_filter( 'embed_oembed_html', array( 'Automattic\Jetpack\VideoPress\Initializer', 'video_enqueue_bridge_when_oembed_present' ), 10, 4 );
+	}
+
+	/**
+	 * Enqueues VideoPress token bridge when a VideoPress oembed is present on the current page.
+	 *
+	 * @param string|false $cache   The cached HTML result, stored in post meta.
+	 * @param string       $url     The attempted embed URL.
+	 * @param array        $attr    An array of shortcode attributes.
+	 * @param int          $post_ID Post ID.
+	 *
+	 * @return string|false
+	 */
+	public static function video_enqueue_bridge_when_oembed_present( $cache, $url, $attr, $post_ID ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		if ( preg_match( '/https?:\/\/(video.wordpress.com|videopress.com)\/(v|embed)\//', $url ) ) {
```

Comment:
Don`t we need to add support to `v.wordpress.com` as well?

Reply:
I don't think I've ever seen `v.wordpress.com` actually used anywhere as an embed, and that url doesn't seem to work for viewing videos on its own. Its possible it was an old way of accessing videos.

I suppose I could add it for completeness.

Reply:
I added a 2nd condition for the `https://v.wordpress.com/GUID` embeds. Since the regex just translates them into a standard embed it was easy to test as well, even though the url itself doesn't do anything.